modules/apps/jaggery/emm/src/emm/units/appbar/public/js/appbar.js

```diff
@@ -77,7 +77,7 @@ function loadNotifications(){
     var serviceURL = "/mdm-admin/notifications/NEW";
```

Comment:
gave that status as a "NEW by backend service",therefore without processing the url to uppercase simply use it as /mdm-admin/notifications/NEW